reference/shipping.v2.yml

```
@@ -1904,6 +1904,32 @@ paths:
         
         Royal Mail has no connection object properties.
         
+        ### Shipping Provider API
+        
+        Please note that this endpoint is not intended for adding connection settings. 
+        
+        Its purpose is solely to establish a connection between your BigCommerce store and the carrier. If the carrier does not have any connection settings configured, please leave the connection property empty. However, if the carrier already has connection settings configured, you can utilize this endpoint to establish the connection using the existing connection properties.
+        
+        PUT or POST example request when your carrier IS NOT configured with any connection settings:
+        
+        ```json
+        {
+          "carrier_id" : "carrier_{your_carrier_id}",
+          "connection" : {}
+        }
+        ```
+        
+        PUT or POST example request when your carrier IS configured with connection settings:
```

```suggestion
        PUT or POST example request when your carrier is configured with connection settings:
```

🍻 I think this would be a little easier to parse without the capitalisation. Same story for `IS NOT` above.

```suggestion
        PUT or POST example request when your carrier has been configured with connection settings:
```

🍹"has been" and "has not been" instead?